lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -43,34 +42,36 @@ def initialize(term_config)
       def find(id, language: nil, replacements: {}, subauth: nil, jsonld: false)
         raise Qa::InvalidLinkedDataAuthority, "Unable to initialize linked data term sub-authority #{subauth}" unless subauth.nil? || term_subauthority?(subauth)
         @language = language_service.preferred_language(user_language: language, authority_language: term_config.term_language)
-        url = authority_service.build_url(action_config: term_config, action: :term, action_request: normalize_id(id), substitutions: replacements, subauthority: subauth, language: @language)
+        @id = id
+        url = authority_service.build_url(action_config: term_config, action: :term, action_request: normalize_id, substitutions: replacements, subauthority: subauth, language: @language)
         Rails.logger.info "QA Linked Data term url: #{url}"
         load_graph(url: url)
         return "{}" unless full_graph.size.positive?
         return full_graph.dump(:jsonld, standard_prefixes: true) if jsonld
-        parse_term_authority_response(id)
+        filter_graph
+        results = map_results
+        convert_results_to_json(results)
```

Comment:
Based on single responsibility principle and to make consistent with the calls used in searchquery.rb, the mapping and json conversion happen in separate methods.